windows/installer.nsi

```diff
@@ -212,6 +218,34 @@ Section "Install_64Bit" INST_64
                 MessageBox MB_OKCANCEL|MB_ICONEXCLAMATION "$(OLD_VER_REMOVE_FAILED)" /sd IDCANCEL IDOK continueinstall
                 goto quit
             ${EndIf}
+
+            ; ask to move old ASIO4ALL registry configuration
+
+            MessageBox MB_YESNO|MB_ICONEXCLAMATION "If your're using ASIO4ALL: Do you want to copy ASIO4ALL settings from the previous version if possible? We will do this for every user." /sd IDYES IDNO continueinstall
```

Comment:
Do we need this message?

Reply:
I can't think of any reason why users would not want it.
As a user, I'd expect Jamulus+ASIO4ALL to work the same after the update than before. This logic tries to do that.
The logic is supposed to be non-destructive, i.e. it should not overwrite any configs (we should be really sure about that).

So my tendency would be to not add a (possibly confusing) message. Would also have the benefit of not needing another translators ping...

Reply:
Ok. Just did a force push without this change. I think now we'll just need to do some tests.